Packages/io.chainsafe.web3-unity/Runtime/Scripts/Connection/ServiceAdapters/EventsServiceAdapter.cs

```diff
@@ -8,14 +8,22 @@ namespace ChainSafe.Gaming.UnityPackage.Connection
 {
     public class EventsServiceAdapter : MonoBehaviour, ILightWeightServiceAdapter
     {
+        [SerializeField, Tooltip("Enable this if a WebSocket connection is unavailable or if the blockchain does not provide a WebSocket URL")]
+        private bool forceEventPolling;
```

Comment:
Question: Why do we make these serialized fields private? I don't think it makes it more safe to use, but it does make it impossible for devs to set this value programmatically.

Don't get me wrong there are cases where we'd want to make the value only editable through the editor ui, but I don't think this is one of those cases.

Reply:
But this is only the 'facade' that is used to inject the event service adapter.

The ForceEVentPolling is public in the config itself, so devs can _always_ inject the event service manually and configure it however they want. They shouldn't seek for the EventsServiceAdapter to do that type of configuration at all, if they're doing everything 'by hand'.